app/views/charge_types/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<% if can?(:read, @charge_type) %>
+  <%- model_class = ChargeType -%>
+  <div class="page-header">
+    <h1><%=t '.title', :default => model_class.model_name.human %></h1>
+  </div>
+
+  <dl class="dl-horizontal">
+    <dt><strong><%= model_class.human_attribute_name(:name) %>:</strong></dt>
+    <dd><%= @task.name %></dd>
```

Comment:
You're referencing tasks here, I assume this is copy pasted from the Task show page?